admin/templates/configuration/tmp/configure/scisql.sh

```diff
@@ -2,30 +2,31 @@
 
 set -e
 
-QSERV_RUN_DIR={{QSERV_RUN_DIR}}
-SCISQL_DIR={{SCISQL_DIR}}
-MYSQL_DIR={{MYSQL_DIR}}
-MYSQLD_SOCK={{MYSQLD_SOCK}}
-MYSQLD_USER={{MYSQLD_USER}}
-MYSQLD_PASS={{MYSQLD_PASS}}
-PYTHON_BIN={{PYTHON_BIN}}
-export PYTHONPATH={{PYTHONPATH}}
+QSERV_RUN_DIR='{{QSERV_RUN_DIR}}'
+SCISQL_DIR='{{SCISQL_DIR}}'
+MYSQL_DIR='{{MYSQL_DIR}}'
+MYSQLD_SOCK='{{MYSQLD_SOCK}}'
+MYSQLD_USER='{{MYSQLD_USER}}'
+MYSQLD_PASS='{{MYSQLD_PASS}}'
+PYTHON_BIN='{{PYTHON_BIN}}'
+export PYTHONPATH='{{PYTHONPATH}}'
 
-PYTHON_DIR=`dirname ${PYTHON_BIN}`
+PYTHON_DIR=$(dirname ${PYTHON_BIN})
 export PATH=${PYTHON_DIR}:${PATH}
-export LD_LIBRARY_PATH={{LD_LIBRARY_PATH}}
+export LD_LIBRARY_PATH='{{LD_LIBRARY_PATH}}'
 
 ${QSERV_RUN_DIR}/etc/init.d/mysqld start
-cd ${SCISQL_DIR}
 echo "-- Deploying sciSQL plugin in MySQL database"
 # password is given on stdin, so that is can't be catched by ps command
 PASSFILE=${QSERV_RUN_DIR}/tmp/pass.txt
 cat <<EOF > ${PASSFILE}
```

Comment:
Do we really need to create this temporary file, can we just pipe password directly to the script?